spec/index/index_spec.rb

```diff
@@ -102,6 +120,11 @@
       subject { Daru::Index.new ('a'..'z').to_a  }
       its(:inspect) { is_expected.to eq "#<Daru::Index(26): {a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t ... z}>" }
     end
+
+    context 'index with name' do
+      subject { Daru::Index.new ['one', 'two', 'three'], name: 'number'  }
+      its(:inspect) { is_expected.to eq "#<Daru::Index: number(3): {one, two, three}>" }
```

Comment:
@zverok ,
I was following same as [dataframe ](https://github.com/SciRuby/daru/pull/323/files#diff-447878c3f829efdf214b2e4db4d1c231L3687)

```
#<Daru::DataFrame: df_name (nrowxncol)
....
```

But `"#<Daru::Index(3): number {one, two, three}>"` is also good.